assembl/views/api2/content_source.py

```diff
@@ -55,6 +61,9 @@ def fetch_posts(request):
             raise HTTPUnauthorized("Only discussion administrator\
                                    can "+'and'.join(requested))
 
+    if force_restart:
```

Comment:
@maparent This will likely require your eyes. We looked into moving the check to `force_restart` and doing the `reset_errors` downstream. We see that it *could* be done in SourceReader class, but we're unsure where would be an ideal place to do a check like this. Can you assist on where would be the best place to put a check like this?